blocks/ads-block/features/ads/default.jsx

```diff
@@ -76,11 +76,10 @@ const ArcAd = (props) => {
 	// istanbul ignore next
 	const isAMP = () => !!(propsWithContext.outputType && propsWithContext.outputType === "amp");
 
-	const [width, height] = config.adClass ? config.adClass.split("x") : [];
+	const [, height] = config.adClass ? config.adClass.split("x") : [];
 
 	const sizing = {
-		maxWidth: `${width}px`,
-		minHeight: reserveSpace ? `${height}px` : null,
+		minBlockSize: reserveSpace && height ? `${height}px` : null,
```

Comment:
If there is no `config.adClass` then the result will be `{ minBlockSize: null }`?

could this be: ```
const sizing = reserveSpace && height ? { minBlockSize: `${height}px` } : {}
```
?

Reply:
The result is `{ minBlockSize: null }`, but React won't render properties into the style attribute that are set to `null`. In this case it also means there'd be no `style` attribute at all since it's the only prop.